erts/doc/src/erlang.xml

```diff
@@ -1970,14 +1972,10 @@ true</pre>
       <fsummary>Check if a function is exported and loaded.</fsummary>
       <desc>
         <p>Returns <c>true</c> if the module <c><anno>Module</anno></c> is
-          loaded and contains an exported function
-          <c><anno>Function</anno>/<anno>Arity</anno></c>,
+          loaded (i.e. not purged or marked as old) and contains an exported
```

Comment:
```suggestion
          loaded (that is, not purged or marked as old) and contains an exported
```

Our documentation guidelines say that we should avoid abbreviations such as "i.e." and "e.g.".